stable/prometheus-nats-exporter/templates/deployment.yaml

```diff
@@ -29,6 +29,11 @@ spec:
             - "-port"
             - "{{ .Values.service.targetPort }}"
             - "-varz"
+            - "-channelz"
```

Comment:
I believe it should be optional. All of there by default.

Reply:
done